pages/index.tsx

```diff
@@ -54,7 +55,8 @@ export default function Home() {
           data.balance - data.stakedBalance - data.unstakedBalance
         ),
         stakedBalance: mutezToTez(data.stakedBalance),
-        unstakedBalance: mutezToTez(data.unstakedBalance)
+        unstakedBalance: mutezToTez(data.unstakedBalance),
+        delegate: data.delegate.address
```

Comment:
if not delegate, will have runtime error. you can use data.delegate?.address

Reply:
Removed it as we dont need it.